python/lsst/ts/scheduler/driver.py

```diff
@@ -92,15 +171,18 @@ def select_next_target(self):
         -------
         Target
         """
+        self.log.log(WORDY, 'Selecting next target.')
+        target = DriverTarget()
 
-        target = Target()
-
-        target.targetid = 0
+        self.targetid += 1
+        target.targetid = self.targetid
```

Comment:
Consider making targetid a constructor argument for DriverTarget, to allow this slightly simpler code and avoid having a target with an invalid ID:
```
self.targetid += 1
target = DriverTarget(targetid = self.targetid)
```